Formula/grin.rb

```diff
@@ -18,8 +18,14 @@ class Grin < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "cdcbd7bcb60160cbd90c54e362650a191b64fb33d356991896c0778ced854ecf"
   end
 
-  depends_on "llvm" => :build # for libclang
+  # Use `llvm@15` to work around build failure with Clang 16 described in
+  # https://github.com/rust-lang/rust-bindgen/issues/2312.
+  # TODO: Switch back to `uses_from_macos "llvm" => :build` when `bindgen` is
+  # updated to 0.62.0 or newer. Check:
+  #   https://github.com/mimblewimble/grin/blob/v#{version}/Cargo.lock
```

Comment:
Can we parse `Cargo.lock` in `install` and then `odie` when this needs to change?

Reply:
That's a good idea!